HouseholdManager/Views/Room/Details.cshtml

```diff
@@ -1,38 +1,46 @@
-﻿@model HouseholdManager.Models.Room
+﻿@model HouseholdManager.Models.ViewModels.RoomDetailViewModel
 @using HouseholdManager.Data.API
 
 @{
-    ViewData["Title"] = "Room Details";
+    ViewData["Title"] =  @Model.RoomName + " Details";
```

Comment:
The @ here breaks the rest of the code block - if you're already in a code block, it isn't needed.
```suggestion
    ViewData["Title"] = Model.RoomName + " Details";
```